docs/en/source/callFlow.rst

```diff
@@ -60,40 +52,41 @@ The Figure is showing the operational flow of entity subscription in the Scorpio
+ 5.2.4. Upon successful subscription condition of subscription request, SM will notify the subscribed entity to the given endpoint back. And also do the remote subscriptions to the context sources provided by the context registry.
 
- 4.3. Upon successful pub operation, SM will send the response back 
-   
-5.SM optionally may raise the query to CR by posting in the CRQuery Topic for each of the subscription requests received (only once per each subscription request). When a message gets published to CRQuery Topic, the consumer CR will be notified who has subscribed or listening on this topic. Now, CR will do the following:
+ 5.3. Upon successful pub operation, SM will send the response back
 
- 5.1. CR will receive the notification and checks for the list of context sources by pulling data from CR Topic and/or CSourceSub Topic for whom this subscription may valid.
+6. SM optionally may raise the query to CR by posting in the CRQuery Topic for each of the subscription requests received (only once per each subscription request). When a message gets published to CRQuery Topic, the consumer CR will be notified who has subscribed or listening on this topic. Now, CR will do the following:
 
- 5.2.CR publishes the list of context sources into the CRQueryResult topic upon which the SM would have already started listening and repeat steps 4.2.3 and 4.2.4.
+ 6.1. Context Source Registry will receive the notification and checks for the list of context sources by pulling data from Registry Topic and/or CSourceSub Topic for whom this subscription may valid.
 
-**Note**: CSource Topic will contain the list of context sources registered through Csource registration interface directly. CR Topic will contain the map of Entity Data model (maintained as an entity ID) created based on entity creation request (through IoT broker interface) and/or provider/data source of that entity model. 
-Limitation: In the first release of Scorpio Broker, Csource query is not supported instead csource query is based on the internal messaging queue mechanism. In the future, both the message queue and Rest based csource query would be supported.
+ 6.2. Context Source Registry publishes the list of context sources into the CRQueryResult topic upon which the SM would have already started listening and repeat steps 5.2.3 and 5.2.4.
+
+**Note**: CSource Topic will contain the list of context sources registered through Csource registration interface directly. Registry Topic will contain the map of Entity Data model (maintained as an entity ID) created based on entity creation request (through IoT broker interface) and/or provider/data source of that entity model. Limitation: In the first release of Scorpio Broker, Csource query is not supported instead csource query is based on the internal messaging queue mechanism. In the future, both the message queue and Rest based csource query would be supported.
```

Comment:
[Major] Please changes in flow diagram 2
- Remove CR Query Result topic and CR Query topic direction
- Remove CSource topic direction
- Change Registry topic direction same as above
- Add direction from I_REGSUB topic to registration subscription manager
- Add direction from  I_NOTIFY topic to subscription manager
-  Add direction from ENTITY topic to subscription manager
-  Add changes flow diagram theoretical part.

Reply:
Done